src/modules/dashboard/service/dashboardService.ts

```diff
@@ -2568,5 +2582,28 @@ export class DashboardService {
     static convertToExactNativeAmount(amount: number){
         return amount > 0 ? amount / Math.pow(10, nativeTokenDivisibility.value) : 0;
     }
+
+    static displayOtherAsset = async(amount: number, assetId: string, mosaic_id: MosaicId) => {
+        // let apiEndpoint = ChainUtils.buildAPIEndpoint(networkState.selectedAPIEndpoint, networkState.currentNetworkProfile.httpPort);
+        // let chainAPICall = new ChainAPICall(apiEndpoint);
+        // let asset = await chainAPICall.assetAPI.getMosaic(mosaic_id);
+
+        // let assetarray = []
+        // assetarray.push(mosaic_id);
+        // let nsAsset = await chainAPICall.assetAPI.getMosaicsNames(assetarray);
+
+        let otherAsset = new OtherAsset();
```

Comment:
no sure what you want to do, but you do not have constructor for OtherAsset

Reply:
Basically for class declaration. Half done subject to your rewrite of dashbaord service

Reply:
then it do not need to be async